Pipfile

```
@@ -59,7 +59,7 @@ werkzeug = "~=2.3"
 prometheus-client = "==0.14.1"
 gds-metrics = {version = "==0.2.4", ref = "6f1840a57b6fb1ee40b7e84f2f18ec229de8aa72", git = "https://github.com/alphagov/gds_metrics_python.git"}
 packaging = "==21.3"
-notifications-utils = {editable = true, ref = "main", git = "https://github.com/GSA/notifications-utils.git"}
+notifications-utils = {editable = true, ref = "notify-512", git = "https://github.com/GSA/notifications-utils.git"}
```

Same note as the other PR, once https://github.com/GSA/notifications-utils/pull/39 is merged!

```suggestion
notifications-utils = {editable = true, ref = "main", git = "https://github.com/GSA/notifications-utils.git"}
```